src/pydata_sphinx_theme/assets/styles/abstracts/_links.scss

```diff
@@ -240,7 +239,7 @@ corresponds to the section in the docs that the user is currently reading.
     &:focus-visible {
       box-shadow: none; // override Bootstrap
       outline: 3px solid var(--pst-color-accent);
-      outline-offset: 3px;
+      outline-offset: 2px;
```

Comment:
I changed the offset here to so that the focus ring would not go into the interior of the hover ring of the dropdown. 

I did, however, decide to let their edges overlap, like so:

![screenshot](https://github.com/pydata/pydata-sphinx-theme/assets/317883/ba0feb30-0add-430d-b1a9-35a3ee3391b3)